common-design/sass/cd-header/_cd-nav.scss

```diff
@@ -29,7 +29,7 @@
     z-index: 101;
     color: $cd-dark-text-color;
 
-    &:after {
+    &:before {
```

Comment:
To avoid conflict in Drupal, when we use the :after pseudo element to set the dropdown arrow icons.
Related to styleguide commit https://github.com/UN-OCHA/ocha_basic/pull/10/commits/248096d6690070b1c2642cc8cc7284558d65510f#diff-aada93b17650fcbc484f9adf1595719aR12